src/test/com/intellij/lang/jsgraphql/ide/editor/GraphQLIntrospectionServiceTest.java

```diff
@@ -18,6 +19,23 @@ public void testPrintIntrospectionJsonAsGraphQL() {
         doTest("schema.json", "schema.graphql");
     }
 
+    public void testPrintIntrospectionJsonAsGraphQL_withEmptyErrors_settingDisabled() {
+        try {
+            GraphQLSettings.getSettings(getProject()).setTreatEmptyErrorsAsSuccessful(false);
+            doTest("schemaWithEmptyErrors.json", "schema.graphql");
+        } catch (IllegalArgumentException exception) {
+            Assert.assertEquals("Introspection query returned an empty errors object. Although it is out of spec, if this behavior is intended, this error can be ignored in the plugin settings.", exception.getMessage());
+            return;
+        }
+
+        throw new RuntimeException("Expected to receive IllegalArgumentException.");
+    }
+
+    public void testPrintIntrospectionJsonAsGraphQL_withEmptyErrors_settingEnabled() {
```

Comment:
Pls, note that we don't use this code style either in the existing code in this repository or in the IntelliJ IDEA sources.

Reply:
Updated test case styling to be more in-line with the other tests.